src/sc_io.h

```diff
@@ -562,6 +571,17 @@ void                sc_io_read (sc_MPI_File mpifile, void *ptr,
                                 size_t zcount, sc_MPI_Datatype t,
                                 const char *errmsg);
 
+/** Check for restricted usage of \ref sc_io_read_at.
+ *
+ * \return              0 if the restriction described in the note of \ref
+ *                      sc_io_read_at applies. This is equivalent to MPI I/O
```

Comment:
Would you be self-consistent here and briefly but completely state the requirement, so the reader does not have to go looking for the original definition?